pkg/containerd/resolver/fetcher.go

```diff
@@ -114,6 +116,30 @@ func (r dockerFetcher) open(ctx context.Context, u, mediatype string, offset int
 		}
 		return nil, errors.Errorf("unexpected status code %v: %v", u, resp.Status)
 	}
+	if offset > 0 {
+		cr := resp.Header.Get("content-range")
+		if cr != "" {
+			if !strings.HasPrefix(cr, fmt.Sprintf("bytes %d-", offset)) {
+				return nil, errors.Errorf("unhandled content range in response: %v", cr)
+
+			}
+		} else {
+			// TODO: Should any cases where use of content range
```

Comment:
todo format..

Reply:
This is copied from the original containerd fix. Wanna to keep change minimal so as to make it easier to maintain the fork or consolidate in the future.

Reply:
yes I saw it was a copy.. todo comment would not be much of a merge :-) but already approved skip the nit .. merge away!